src/helper.jl

```diff
@@ -1,6 +1,19 @@
 using Distributions
 
-function loadparameters(datadir=joinpath(dirname(@__FILE__), "..", "data"))
+"""
+Converts a year value into an integer corresponding to fund's time index.
+"""
+function getindexfromyear(year)
+    const baseyear = 1950
+    return Int(year - baseyear + 1)
```

Comment:
Curious why the `Int()` is required here. Can year be something other than an `Int`? If so, what else would it be that can be converted to an `Int` (i.e., not a float)? Could declare `year::Int` if that's really the case.

Reply:
I was using years as scenario arguments to the monte carlo simulation, and although I was defining them as Ints: 
```
scenario_args = [:emissionyear => [2020, 2050]]
```
When I later unpack the scenario tuple from within the scenario function, the emissionyear gets returned as a Float, which then when passed to `getindexfromyear` gives a Float which doesn't work as an index. Is there something I should be doing in the scenario_args definition to specify that I need these values to be Ints?

Reply:
This seems like something that should be fixed to not require type conversion. I'll add an issue for it. As a reminder in the meantime, we might do:
```
function getindexfromyear(year::Int)
    const baseyear = 1950
    return year - baseyear + 1

# TBD: remove this when issue of int-to-float conversion is resolved
getindexfromyear(year) = getindexfromyear(Int(year))
```